doc/installationguide.md

```diff
@@ -278,6 +278,8 @@ used for the same purpose. For instance:
     standard, but the final decision has yet been confirmed), take into account it could change
 -   **explicitAttrs**: if this flag is activated, only provisioned attributes will be processed to Context Broker. This
     flag is overwritten by `explicitAttrs` flag in group or device provision.
+-   **defaultEntityNameConjunction**: the default conjunction string used to compose a default entity_name when is not 
+    provided at device provisioning time. This value is overwritten by `defaultEntityNameConjunction` in group provision.
```

Comment:
> ... This value is overwritten by `defaultEntityNameConjunction` in group provision

Two things:

1. Just to confirm the field can be used only in group provision but not in device provision
2. The new field `defaultEntityNameConjunction` should be added to the group provision documentation (not sure in which .md is... I refer to the table we have with all possible fiels).

Reply:
better explained by https://github.com/telefonicaid/iotagent-node-lib/pull/949/commits/d870b94d738aa99436852ca730f40bf4becb4f1c

Reply:
>   ... (not sure in which .md is... I refer to the table we have with all possible fields)

I mean this: https://github.com/telefonicaid/iotagent-node-lib/blob/master/doc/api.md#service-group-model

Reply:
fixed in https://github.com/telefonicaid/iotagent-node-lib/pull/949/commits/2321e665aef37d5824bba499168319002dd595b4